File: app/src/main/java/com/example/prashant/databinding/di/module/FragmentModule.java
package com.example.prashant.databinding.di.module;

import com.example.prashant.databinding.ui.ListItemFragment;

import dagger.Module;
import dagger.android.ContributesAndroidInjector;

@SuppressWarnings("unused")
@Module
public abstract class FragmentModule {
    @ContributesAndroidInjector
    public abstract ListItemFragment contributeListFragment();
}

File: app/src/main/java/com/example/prashant/databinding/di/module/FragmentModule.kt
package com.example.prashant.databinding.di.module

import com.example.prashant.databinding.ui.ListItemFragment

import dagger.Module
import dagger.android.ContributesAndroidInjector

@Module
abstract class FragmentModule {
    @ContributesAndroidInjector
    abstract fun contributeListFragment(): ListItemFragment
}
